Add a document for running the 2026 Project for the first time (#12)
oaklandrobotics/docs: src/projects/igvc-2026/software/environment-setup/index.md

Read the official documentation [here](https://docs.ros.org/en/jazzy/).

### Installation methods

ROS 2 is supported on multiple platforms, but we recommend either using a native installation of Ubuntu or a containerized Docker environment, as these approaches are the most stable and easiest to support as a team. You can reference the official documentation (linked above) for alternative installation options.

> Our recommended Ros 2 setup paths:

Reviewer: ```suggestion
Our recommended ROS 2 setup paths:
```

**Rollback safety** | Can reset back to default state at any time | Can be *technically* be done, but more effort
**Debugging** | Extra layer to access tools, files, logs | Direct access to access tools, files, logs

See (or message) Kayla Dawkins (@danielle217) if you have any questions about the benefits of Docker or issues with the setup

## Setting up your workspace

> Your workspace in Ros 2 refers to your local folder 

Reviewer: ```suggestion
A workspace in ROS 2 refers to the directory which contains the ROS 2 packages used in your project. All of the code that we write will be contained inside of a package in the workspace.
```

> which contians the robotics project itself.

Reviewer: ```suggestion

```
